### PR TITLE
16.11 release note update

### PR DESCRIPTION
**Pull request by rainersigwald**
- 16.10.2 release notes
- Update relnotes for 16.11

### CODE REVIEW

File: documentation/Changelog.md
Comment (Forgind) [inline documentation/Changelog.md:47]: nit:
```suggestion
* Fixed a regression in glob matching where files without extensions were erroneously not matched (#6531).
```
?

**Review by Forgind — APPROVED** [review]

**Comment by BenVillalobos** [comment]
I plan on merging this with #6656 to avoid getting 2 QB approvals before GA

**Comment by rainersigwald** [comment]
@BenVillalobos this doesn't really have to go into 16.11 at all, just "somewhere". So it can go to main after 16.11 RTM.

**Comment by BenVillalobos** [comment]
Cherry-picked / squashed onto https://github.com/dotnet/msbuild/pull/6656

**Comment by BenVillalobos** [comment]
> this doesn't really have to go into 16.11 at all, just "somewhere". So it can go to main after 16.11 RTM.

Did not read this before closing / cherry-picking. I'll stick it in vs16.11 since the work is done in that PR anyway. Reopening to get this into main docs eventually.

**Comment by Forgind** [comment]
Anything you want to add before merging this? Want me to make some release notes for 17.0?

**Comment by rainersigwald** [comment]
> Anything you want to add before merging this?

Yeah, I completed the 16.11 notes.

> Want me to make some release notes for 17.0?

Go for it.

**Comment by rainersigwald** [comment]
Pushed https://github.com/dotnet/msbuild/releases/tag/v16.11.0 with the current contents.

File: documentation/Changelog.md
Comment (eriawan) [inline documentation/Changelog.md:5]: NIT: I think it should be `is shipped` or for better active instead of passive, should be rephrased using `ships` as:
`This version of MSBuild ships with Visual Studio 2019 version 16.11.0 and .NET SDK 5.0.400.`

File: documentation/Changelog.md
Comment (eriawan) [inline documentation/Changelog.md:47]: Same as line 4 comment, should be `is shipped` unless it is emphasizing on the past state of 16.10.2

**Review by eriawan — COMMENTED** [review]
Just a small feedback :)

File: documentation/Changelog.md
Comment (rainersigwald) [reply documentation/Changelog.md:5]: I prefer past tense here because this release is in some ways just a pointer to the "standard" ways to get MSBuild: through VS or the .NET SDK.

Comment (eriawan) [reply documentation/Changelog.md:5]: @rainersigwald 
thanks for your reply! ok, I understand now 🙂

**Pull request merged**
